src/doc/trpl/iterators.md

```diff
@@ -4,20 +4,20 @@ Let's talk about loops.
 
 Remember Rust's `for` loop? Here's an example:
 
-```{rust}
-for x in range(0, 10) {
+```
+for x in 0..10 {
     println!("{}", x);
 }
 ```
 
-Now that you know more Rust, we can talk in detail about how this works. The
-`range` function returns an *iterator*. An iterator is something that we can
+Now that you know more Rust, we can talk in detail about how this works.
+Ranges (the `0..10`) are 'iterators'. An iterator is something that we can
 call the `.next()` method on repeatedly, and it gives us a sequence of things.
 
 Like this:
 
-```{rust}
-let mut range = range(0, 10);
+```rust
```

Comment:
The convention I use is to add `rust` for non-API docs, so that GitHub knows how to highlight them.